test/e2e/lib/browser-manager.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * @file Manages instance of Playwright Browser and BrowserContext.
+ * @author Edwin Takahashi
+ */
+
+/**
+ * External dependencies
+ */
+import { Browser, BrowserContext, chromium } from 'playwright';
+import config from 'config';
+
+const browserStartTimeoutMS = 2000;
+
+export let browser: Browser;
+
+/**
+ * Returns the target screen size for tests to run against.
+ *
+ * By default, this function will return 'desktop' as the target.
+ * To specify another screen size, set the BROWSERSIZE environment variable.
+ *
+ * @returns {string} String representation of the target screen size.
+ */
+export function targetScreenSize(): string {
```

Comment:
If we know what the other screen sizes are allowed to be, it might be nice to specify them inline like `(): 'desktop' | 'mobile'`

This might also be useful for targetLocale (perhaps a type already exists for those)

Reply:
I suppose this comment is covering [this section](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/typescript.md#usage-patterns) in the TypeScript guidelines?

In this case, the return value defaults to `desktop`, but it could be any of `laptop, mobile, desktop or tablet`. How should I handle this case?

The same would go for locale as well. It defaults to `en`, but it could be any one of the multiple strings we support.

Reply:
I've pushed a new revision that removes usage of primitives for the `screenSize` and `localeCode` related functions. Can you take a look and let me know if I'm on the right path, thanks!